search-sentiments-map/index.yaml

```diff
@@ -2,6 +2,7 @@ indexes:
 
 - kind: WorldDataByTopic
   properties:
+  - name: topic
```

Comment:
Cuz we are also filtering by topic in some case

Reply:
But this may break it. If you are filtering by a different item it needs a separate index section. Also, I am not sure where we filter by topic, I thought we had changed all filter by topics to filter by lowercase topic.

Reply:
In search.js `retrieveSearchResultFromDatastore` we filter by topic.
I'm writing two separate ones. I have not deployed this so shouldn't have broken anything

Reply:
If you deploy it and it doesnt work, it should give you an error message and prevent you from finishing the deploy, so don't worry you will know if it is broken.

Reply:
The two indices already exist in the datastore so if we don't deploy index.yaml it should be fine lol